src/nxdoc/web-ui/nuxeo-elements/notify-behavior.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Notify behavior
+description: 'Documentation for the notify behavior'
+review:
+    date: '2021-09-02'
+    status: ok
+toc: true
+labels:
+    - lts2021-ok
+    - nuxeo-elements
+    - nuxeo-web-ui
+    - polymer
+    - behavior
+    - rdias
+tree_item_index: 300
+---
+
+In [ELEMENTS-1368](https://jira.nuxeo.com/browse/ELEMENTS-1368) we introduced the `nuxeo-notify-behavior`, as an effort to standardize the way our custom elements send notifications upwards in the DOM.
+
+Before the introduction of this behavior, elements would fire a `notify` event via `dispatchEvent` to report information to parent elements, however, this strategy was faulty when the element was not connected to the DOM (e.g., if it was hidden either by a dom-if or a nuxeo-filter).
+
+To address this problem, the new behavior exposes a `notify` method that fires a the `notify` event from:
+- the current element, if it is attached to the DOM
+- from a fallback target element if the current element is disconnected
+
+The fallback element can be set with a helper method and it can be used at the application level. 
+
+On Nuxeo Web UI, the fallback element is the `nuxeo-app` (see [WEBUI-345](https://jira.nuxeo.com/browse/WEBUI-345)).
+
+## How to use the behavior
+
+To use the `nuxeo-notify-behavior` in your custom elements, import the behavior with:
+
+{{#> panel type='code'}}
+```js
+import { NotifyBehavior } from '@nuxeo/nuxeo-elements/nuxeo-notify-behavior.js';
+```
+{{/panel}}
+
+Next, you just need to call the `notify` method with the required event `options` object (depends on the element handling the event):
+
+{{#> panel type='code'}}
+```js
+_customElementMethod() {
+  this.notify({ message: 'A custom message.' });
```

Comment:
I think we could also take this opportunity to explain the other options: abort, close, dismissible, duration. If one of these was added during a patch, we could leave a note pointing to the version and the ticket. WDYT?

Reply:
It sounds perfect, I am adding a new entry here. Let me know what you think about it.